src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -1074,12 +379,12 @@ namespace JSONHelpers
             DeviceInfoJSON result;
 
             result.deviceId = device.GetNamedString(L"device-id");
-            if (!isValidDeviceId(result.deviceId))
+            if (!IsValidDeviceId(result.deviceId))
             {
                 return std::nullopt;
             }
 
-            if (auto zoneSet = ZoneSetData::FromJson(device.GetNamedObject(L"active-zoneset")); zoneSet.has_value())
+            if (auto zoneSet = JSONHelpers::ZoneSetDataJSON::FromJson(device.GetNamedObject(L"active-zoneset")); zoneSet.has_value())
```

Comment:
Replace all strings literals with const strings and explicitly mark them as non-localizable.